CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/GiniConfiguration.swift

```diff
@@ -529,6 +529,11 @@ import UIKit
       * Set an adapter implementation to show a custom bottom navigation bar on the camera screen.
      */
     public var cameraNavigationBarBottomAdapter: CameraBottomNavigationBarAdapter?
+
+    /**
+      * Set an adapter implementation to show a custom bottom navigation bar on the review screen.
+     */
+    public var reviewNavigationBarBottomAdapter: ReviewScreenBottomNavigationBarAdapter?
```

Comment:
Thank you @chriskryn for bringing my attention.
[Suggection]@vizaknaidavid could you please create a xib for this bottom navigation - it will remove some constraints from the code?